src/query/storages/parquet/src/parquet_rs/parquet_table/meta.rs

```diff
@@ -39,6 +39,7 @@ pub async fn read_metas_in_parallel(
     leaf_fields: Arc<Vec<TableField>>,
     num_threads: usize,
     max_memory_usage: u64,
```

Comment:
This doesn't work:)
 We have many write query like:copy/replace/insert/merge into from (select * from @stage/parquet), they are all can read stage parquet files.
For mostly users, write is one query and read is another, so if we cache in the write side, there is no use.

This PR aims to fast read when read a remote parquet many times, but it's not a normal case in facts.